config.toml.example

```diff
@@ -411,24 +411,16 @@ changelog-seen = 2
 #
 # Note that debuginfo-level = 2 generates several gigabytes of debuginfo
 # and will slow down the linking process significantly.
-#
-# Defaults to 1 if debug is true
-#debuginfo-level = 0
+#debuginfo-level = debug (int)
```

Comment:
debug is not an int, so this is a bit confusing. I think the old = 0 and comment was better.